APE8.rst

```diff
@@ -0,0 +1,94 @@
+Astropy project (and other projects of similar mind).  This APE provides the
+text of just such a code of conduct, which will be adopted by the project to
+provide guidance to the community. The code of conduct will be licensed using
+the Creative Commons Attribution license, allowing other projects to use this
+code or adapt it to their needs.
+
+
+Detailed description
+--------------------
+
+The text of the code of conduct is shown here:
+
+Astropy Community Code of Conduct
+=================================
+
+The community of participants in open source Astronomy projects is made up of members from around the globe with a diverse set of skills, personalities, and experiences. It is through these differences that our community experiences success and continued growth. We expect everyone in our community to follow these guidelines when interacting with others both inside and outside of our community. Our goal is to keep ours a positive, inclusive, successful, and growing community.
+
+As members of the community,
+
+* We pledge to treat all people with respect and provide a harassment- and bullying-free environment, regardless of sex, sexual orientation and/or gender identity, disability, physical appearance, body size, race, nationality, ethnicity, and religion. In particular, sexual language and imagery, sexist, racist,  or exclusionary jokes are not appropriate.
+* We pledge to respect the work of others by recognizing acknowledgment/citation requests of original authors. As authors, we pledge to be explicit about how we want our own work to be cited or acknowledged.
+* We pledge to welcome those interested in joining the community, and realize that including people with a variety of opinions will only serve to enrich our community. In particular, discussions relating to pros/cons of various technologies, programming languages, and so on are welcome, but these should be done with respect, taking proactive measure to ensure that all participants are heard and feel confident that they can freely express their opinions.
+* We pledge to welcome questions and answer them respectfully, paying particular attention to those new to the community. We pledge to provide respectful criticisms and feedback in forums, especially in discussion threads resulting from code contributions.
+* We pledge to be conscientious of the perceptions of the wider community and to respond to criticism respectfully. We will strive to model behaviors that encourage productive debate and disagreement, both within our community and where we are criticized. We will treat those outside our community with the same respect as people within our community.
+* We pledge to help the entire community follow the code of conduct, and to not remain silent when we see violations of the code of conduct. We will take action when members of our community violate this code such as contacting community@astropy.org or talking privately with the person.
```

Comment:
Good point about the email address. How about actually having an email address called confidential@astropy.org and then also say in parentheses (all emails sent to this address will be treated with the strictest confidence)